packages/gatsby/cache-dir/gatsby-browser-entry.js

```diff
@@ -29,6 +29,24 @@ const StaticQuery = props => (
   </StaticQueryContext.Consumer>
 )
 
+const useStaticQuery = query => {
+  if (
+    typeof React.useContext !== `function` &&
+    process.env.NODE_ENV === `development`
+  ) {
+    throw new Error(
+      `You're likely using a version of React that doesn't support Hooks` +
```

Comment:
@m-allanson Can you help me write a better error message here please?

Reply:
I'm pretty OK with our error message here! Think it's OK.

Maybe I'd add to update React _and_ ReactDOM? It could also be useful _how_ to update both?

Reply:
Added the point about ReactDOM! Instructions for update will have to take into account npm or yarn etc so leaving that out